_posts/2020-10-29-cockpit-231.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 231
+author: marusak
+date: 2020-10-29
+tags: cockpit linux
+slug: cockpit-231
+category: release
+summary: Cockpit with new performance graphs
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 231.
+
+### Replace system's graph page with a completely new USE method page
+
+System graphs (reachable after clicking on 'View graphs' on the overview page) were replaced with
+a dedicated page for the analyzing current and the historical performance of a machine.
+This page (reachable after clicking on 'View details and history' on the overview page)
+shows current metrics as well as historical events based on the
+[USE](http://www.brendangregg.com/usemethod.html) method.
```

Comment:
I don't think we need to be extra verbose about the old page, it just looks like this is being written twice. Also fixing some grammar issues.

```suggestion
The system graphs were replaced with a dedicated page for analyzing the current and historical performance of a machine. This page (reachable after clicking on 'View details and history' on the Overview page) shows current metrics as well as historical events based on the [USE](http://www.brendangregg.com/usemethod.html) method.
```